src/patternfly/components/Table/table-tree-view.scss

```diff
@@ -0,0 +1,52 @@
+$pf-c-tree-view--MaxDepth: 10;
+
+.pf-c-table {
+  // main
+  --pf-c-table__tree-view-main--GridTemplateColumns: var(--pf-c-table--m-tree-view__toggle--Width) max-content 1fr;
+  --pf-c-table__tree-view-main--indent--base: var(--pf-global--spacer--lg);
+  --pf-c-table__tree-view-main--MarginLeft: calc(var(--pf-global--spacer--md) * -1); // based off of the expected width of the toggle button
+
+  // check
+  --pf-c-table__tree-view-main--c-table__check--MarginRight: var(--pf-global--spacer--md);
+
+  // icon
+  --pf-c-table__tree-view-icon--MinWidth: var(--pf-global--FontSize--md);
+  --pf-c-table__tree-view-icon--MarginRight: var(--pf-global--spacer--sm);
+
+  // toggle
+  --pf-c-table--m-tree-view__toggle--Width: #{pf-size-prem(42px)}; // based off of the expected width of the toggle button
```

Comment:
The react button will be different. The core fa icon is 10px, so 16px (button padding) * 2 + 10px is 42px. The react icon will be 48px since the icon is 16px wide. Should we do this the way we do it in tree view and dual list selector, where nested tree view items have consistent padding, and the toggle button is absolutely positioned (to take it out of the layout) and `translateY()`'d to the left 100%? That will maintain aligned nested content, and allow for minor variation in toggle width.